src/core/vr-scene.js

```diff
@@ -7,6 +7,11 @@
         value: function() {
           this.attachEventListeners();
           this.setupScene();
+
+          // handle fullscreen changes
+          document.addEventListener('fullscreenchange', this.fullscreenChange.bind(this), false);
+          document.addEventListener('mozfullscreenchange', this.fullscreenChange.bind(this), false);
+          document.addEventListener('webkitfullscreenchange', this.fullscreenChange.bind(this), false);
```

Comment:
Or even better. create a function attachFullscreenListeners that it's called from attachEventListeners